pyspark/bigdl/examples/dlframes/ImageInference/ImageInferenceExample.py

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.sql.functions import col, udf
+from pyspark.sql.types import StringType
+from bigdl.util.common import *
+from bigdl.dlframes.dl_image_reader import *
+from bigdl.dlframes.dl_image_transformer import *
+from bigdl.transform.vision.image import *
+from bigdl.dlframes.dl_classifier import *
+from bigdl.nn.layer import Model
+
+if __name__ == "__main__":
+
+    if len(sys.argv) != 3:
```

Comment:
Use argparse or OptionParser to make it more clear to users ?

Reply:
Thanks. I noticed that many of our examples are using OptionParser. Yet I tested and found it misleading.
When user input is invalid, OptionParser cannot print a helpful information, and argparse marks all the parameters as optional.

Since both of the parameters are required and do not have default values, I'd prefer to use <> around them.